src/types/personalRoomsTypes.ts

```
@@ -25,10 +38,10 @@ export type PersonalRoomsDocumentProps = {
     Apartment: Omit<IApartment, "Rooms"> & {
         Rooms: RoomsType[];
     };
-    TailorShop: Omit<ITailorShop, "Rooms"> & {
+    TailorShop: Omit<ITailorShopDatabase, "Rooms"> & {
         Rooms: RoomsType[];
     };
 };
 
 // eslint-disable-next-line @typescript-eslint/ban-types
-export type PersonalRoomsModelType = Model<IPersonalRooms, {}, PersonalRoomsDocumentProps>;
+export type PersonalRoomsModelType = Model<IPersonalRoomsDatabase, {}, PersonalRoomsDocumentProps>;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)